### PR TITLE
Use AWS SES instead of SMTP

### PR DESCRIPTION
**Pull request by jsf9k**
Use AWS SES to send email messages (via [boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/quickstart.html)) instead of connecting directly to an SMTP server.

### CODE REVIEW

File: cyhy/mailer/cli.py [hunk 1]
```diff
@@ -314,13 +273,26 @@ def get_federal_cyhy_requests(db):
     return requests
 
 
-def send_message(mail_server, message, counter=None):
+class UnableToSendError(Exception):
+    """Raised when there an error is encountered when attempting to send
```
Comment (dav3r) [inline cyhy/mailer/cli.py]: Minor typo in comment; remove `there`:   Raised when ~~there~~ an error is encountered...

**Review by dav3r — APPROVED** [review]
Way to simplify! 👍

File: cyhy/mailer/cli.py (hunk 1 above)
Comment (jsf9k) [reply cyhy/mailer/cli.py]: Fixed in d5794dd.

**Comment by jsf9k** [comment]
@cisagov/team-ois Please review so we can get this old PR out the door. :older_man:

**Pull request merged**